### ISSUE TITLE
MLM with mixed effects for categorical outcomes in SPSS

### ISSUE THREAD

**Issue by Khadeejafahad** [jasp-stats/jasp-issues#374]
* Enhancement:
* Purpose:
* Use-case:

**Is your feature request related to a problem? Please describe.**
 

**Describe the solution you'd like**
 

**Describe alternatives you've considered**
 

**Additional context**

**Comment by Khadeejafahad** [comment]
Hello everyone
I joined this platform today after an acquaintance mentioned this to me. 
I have been struggling learn MLM with mixed effect model for categorical outcomes using SPSS. I am unable to find any tutorial for this analysis on YouTube.
I have collected level 1 data (10 answers to a question by every participant), however, level 2 data is computed from proportions of level 1 data. 
Total level 1 responses = 2100
Total participants = 210.
Outcome = years categorized into three groups.
. I would me really really grateful if someone could guide me
